.github/workflows/e2e-test-provider-example.yml

```diff
@@ -186,7 +212,11 @@ jobs:
           fi
 
           # take the middle (2nd) supported Kubernetes version (default)
-          kubernetes_version="$(../build/constellation config kubernetes-versions | awk 'NR==3{print $1}')"
+          if [[ "${{ inputs.providerVersion }}" != "" ]]; then
+            kubernetes_version="$(../release/constellation config kubernetes-versions | awk 'NR==3{print $1}')"
+          else
+            kubernetes_version="$(../build/constellation config kubernetes-versions | awk 'NR==3{print $1}')"
+          fi
 
           cat > _override.tf <<EOF
           terraform {
```

Comment:
nit: the whitespaces in this generated Terraform file are very inconsistent (they were already before this PR)
e.g.
```terraform
terraform {
  required_providers {
    constellation = {
      source  = "edgelesssys/constellation"
      version            = "${version}"
    }
  }
}
```

Reply:
We could easily address this by running terraform fmt ... after writing the file.

Reply:
This was more about the formatting in GitHub actions.
I don't think there is a need to have our e2e actions use a file that has pretty formatting.
But having consistent whitespaces helps with reading the test and editing it in the future.